tests/suites/test_suite_lms.data

```diff
@@ -6,6 +6,7 @@ lms_sign_verify_test:"c41ba177a0ca1ec31dfb2e145237e65b":"626201f41afd7c9af793cf1
 LMS NULL-message sign-verify test
 # This test uses a NULL zero-length message, and then generates a private key,
 # signs the message, and verifies the signature.
+depends_on:!MBEDTLS_VALGRIND
```

Comment:
Why skip the test case for this one, but the test function for the one above? This inconsistency makes it harder to follow what gets executed.